### PR TITLE
Add workaround for Liquibase CORE-2024

### PR DESCRIPTION
**Pull request by seanf**
https://liquibase.jira.com/browse/CORE-2024

### CODE REVIEW

**Comment by seanf** [comment]
@aeng Looks like all the tests passed, and no sign of the `org.jboss.module` warning you found.

File: zanata-war/src/main/java/org/eclipse/jetty/server/handler/AbstractHandler.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.eclipse.jetty.server.handler;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Workaround for unwanted logging caused by Liquibase.
+ * http://stackoverflow.com/questions/26530677/why-failed-to-define-class-warn-from-server-startup.
+ * https://liquibase.jira.com/browse/CORE-2024.
+ *
+ * @author cilap http://stackoverflow.com/a/29942016/14379
```
Comment (aeng) [inline zanata-war/src/main/java/org/eclipse/jetty/server/handler/AbstractHandler.java]: @seanf do you want to have yourself as one of the author too? or not?

Comment (seanf) [inline zanata-war/src/main/java/org/eclipse/jetty/server/handler/AbstractHandler.java]: All I did was delete some `println`s, so I wasn't going to bother.

**Comment by aeng** [comment]
:+1:

**Pull request merged**